Description: Set the language level to 1.8 to fix the build failure with Ant 1.10 and OpenJDK 17
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/subprojects/core/core.gradle
+++ b/subprojects/core/core.gradle
@@ -15,7 +15,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = 1.8
 
 configurations {
     reports
--- a/subprojects/base-services-groovy/base-services-groovy.gradle
+++ b/subprojects/base-services-groovy/base-services-groovy.gradle
@@ -15,7 +15,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api libraries.groovy
--- a/subprojects/base-services/base-services.gradle
+++ b/subprojects/base-services/base-services.gradle
@@ -6,7 +6,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api libraries.guava
--- a/subprojects/build-option/build-option.gradle
+++ b/subprojects/build-option/build-option.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':cli')
--- a/subprojects/cli/cli.gradle
+++ b/subprojects/cli/cli.gradle
@@ -22,6 +22,6 @@
 */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 useClassycle()
--- a/subprojects/core-api/core-api.gradle
+++ b/subprojects/core-api/core-api.gradle
@@ -15,7 +15,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(":baseServices")
--- a/subprojects/dependency-management/dependency-management.gradle
+++ b/subprojects/dependency-management/dependency-management.gradle
@@ -1,6 +1,6 @@
 apply plugin: "groovy"
 
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 configurations {
     mvn3Input
--- a/subprojects/jvm-services/jvm-services.gradle
+++ b/subprojects/jvm-services/jvm-services.gradle
@@ -3,7 +3,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(":baseServices")
--- a/subprojects/language-groovy/language-groovy.gradle
+++ b/subprojects/language-groovy/language-groovy.gradle
@@ -1,6 +1,6 @@
 
 // Compiler daemon
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 dependencies {
     compile project(":core")
--- a/subprojects/language-jvm/language-jvm.gradle
+++ b/subprojects/language-jvm/language-jvm.gradle
@@ -1,4 +1,4 @@
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 dependencies {
     compile project(":core")
--- a/subprojects/launcher/launcher.gradle
+++ b/subprojects/launcher/launcher.gradle
@@ -1,7 +1,7 @@
 import org.gradle.build.GradleStartScriptGenerator
 
 // Main entry point requires Java 5
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 configurations {
     startScriptGenerator
--- a/subprojects/logging/logging.gradle
+++ b/subprojects/logging/logging.gradle
@@ -3,7 +3,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/messaging/messaging.gradle
+++ b/subprojects/messaging/messaging.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/model-groovy/model-groovy.gradle
+++ b/subprojects/model-groovy/model-groovy.gradle
@@ -20,7 +20,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/native/native.gradle
+++ b/subprojects/native/native.gradle
@@ -3,7 +3,7 @@
 */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/platform-play/platform-play.gradle
+++ b/subprojects/platform-play/platform-play.gradle
@@ -1,7 +1,7 @@
 apply plugin: "groovy"
 
 // Compiler daemon
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 dependencies {
     compile project(":core")
--- a/subprojects/plugins/plugins.gradle
+++ b/subprojects/plugins/plugins.gradle
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 configurations {
     testFixtures
--- a/subprojects/process-services/process-services.gradle
+++ b/subprojects/process-services/process-services.gradle
@@ -3,7 +3,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/resources/resources.gradle
+++ b/subprojects/resources/resources.gradle
@@ -3,7 +3,7 @@
  */
 apply plugin: 'java-library'
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     api project(':baseServices')
--- a/subprojects/scala/scala.gradle
+++ b/subprojects/scala/scala.gradle
@@ -15,7 +15,7 @@
  */
 
 // Compiler daemon
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 dependencies {
     compile libraries.groovy
--- a/subprojects/tooling-api/tooling-api.gradle
+++ b/subprojects/tooling-api/tooling-api.gradle
@@ -1,7 +1,7 @@
 import org.gradle.ShadedJar
 
 // GradleConnector entry point requires Java 5
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 configurations {
     testPublishRuntime
--- a/subprojects/workers/workers.gradle
+++ b/subprojects/workers/workers.gradle
@@ -1,5 +1,5 @@
 
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     compile project(':core')
--- a/subprojects/wrapper/wrapper.gradle
+++ b/subprojects/wrapper/wrapper.gradle
@@ -15,7 +15,7 @@
  */
 
 // Wrapper main entry point requires Java 5
-sourceCompatibility = javaVersion.java9Compatible ? 1.6 : 1.5
+sourceCompatibility = javaVersion.java9Compatible ? 1.7 : 1.5
 
 dependencies {
     compile project(":cli")
--- a/subprojects/testing-jvm/testing-jvm.gradle
+++ b/subprojects/testing-jvm/testing-jvm.gradle
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 // Test execution
-sourceCompatibility = 1.6
+sourceCompatibility = 1.7
 
 dependencies {
     compile project(':core')
